README.md

```diff
@@ -44,15 +44,15 @@ Istio is composed of these components:
   > simplifies and enhances how microservices in an application talk to each
   > other over the network provided by the underlying platform.
 
-- **Istiod** - The Istio control plane. It provides service discovery, configuration and certificate management. It consists of the following sub-components:
+- [Istiod](https://istio.io/latest/blog/2020/istiod/) - The Istio control plane. It provides service discovery, configuration and certificate management. It consists of the following sub-components:
```

Comment:
Probably don't want the README (permanent) to link to a blog (somewhat ephemeral/point in time)

Reply:
Removed blog links. Thanks